src/CodeGen_Hexagon.cpp

```diff
@@ -62,11 +62,6 @@ std::unique_ptr<llvm::Module> CodeGen_Hexagon::compile(const Module &module) {
         if (module.target().has_feature(Halide::Target::HVX_64))
```

Comment:
Also, this 'if' should probably just be:

    if (module.target().has_feature(...) && module.target().has_feature(...))

or even

    if (module.target().features_all_of({Target::HVX_64, Target::HVX_128}))